ingest/validation/validate_metadata.py

```diff
@@ -726,10 +727,23 @@ def validate_collected_ontology_data(metadata, convention):
     for entry in metadata.ontology.keys():
         # provision to skip validation of non-EBI OLS metadata for now
         if entry == 'organ_region':
-            continue
+            MBA_url = 'https://raw.githubusercontent.com/broadinstitute/scp-ingest-pipeline/jlc_validate_organ_region/schema/organ_region/mouse_brain_atlas/MouseBrainAtlas.csv'
```

Comment:
Blocker: There's an interesting predicament here.  

**Problem 1**
If someone deletes the innocuously-named `jlc_validate_organ_region` branch, then to my undertanding the current `MBA_url` value will return a 404 and production will break.

One solution would be to update `MBA_url` to point to the `master` branch.

**Problem 2**
Updating to `master` means that your test using this ontology would fail before this branch is merged.

---

So both approaches seem problematic, in different ways, and one problem can't be resolved without causing the other.  We can avoid both by pointing to a particular commit ID, but then we sacrifice the ability to always point to the latest version.

Solving Problem 1 instead of (the merely ceremonial) Problem 2 strikes me as a better trade-off.  For example:

```suggestion
            MBA_url = 'https://raw.githubusercontent.com/broadinstitute/scp-ingest-pipeline/master/schema/organ_region/mouse_brain_atlas/MouseBrainAtlas.csv'
```

Reply:
Thanks for catching this issue, Eric. My current thinking is that this is ontology data and should only change rarely.  It is less important that we point to latest and more important that the code is always pointed to the ontology file that will give correct validation. In that sense, the convention in the repo at this point in time (ie. for commit X) can be tied to an appropriate instance of the ontology file for a pre-existing commit.

Reply:
`bf48ed8` updated to point to first commit where file was added to repo